ui/src/components/easter/homepage/EasterCelebrationHomeContainer.js

```diff
@@ -0,0 +1,154 @@
+import React from 'react';
```

Comment:
![Screenshot 2022-04-11 at 22 27 07](https://user-images.githubusercontent.com/33679483/162761338-969ae3e1-aea3-4e98-b009-1616f2b441dc.png)

at lower/smaller devices (iphoneSE, iphone12, iphoneXR, samsung galaxy) the fonts are a bit too big and the buttons are not immediately visible. probably have to adjust font size and vertical height on these smaller devices